### PR TITLE
[FIX] account_edi_ubl_cii: remove deprecated EAS code for Sweden

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
Rremove deprecated EAS code for Sweden. The code is no longer part of the valid list of codes.

See https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/ and https://docs.peppol.eu/edelivery/codelists/

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
ping @JulienVR @smetl

File: addons/account_edi_ubl_cii/models/account_edi_common.py [hunk 1]
```diff
@@ -77,7 +77,6 @@
     'SM': 9951,
     'TR': 9952,
     'VA': 9953,
-    'SE': 9955,
```
Comment (JulienVR) [inline addons/account_edi_ubl_cii/models/account_edi_common.py:80]: You need to replace the old number by the new one, I believe '0007' ?
Also please retarget 15.0.

Comment (JordiMForgeFlow) [reply addons/account_edi_ubl_cii/models/account_edi_common.py:80]: yes you are right, it is 0007

**Comment by JordiMForgeFlow** [comment]
@JulienVR I have opened 15.0 PR here: https://github.com/odoo/odoo/pull/153440

**Comment by JulienVR** [comment]
closed in favor of: https://github.com/odoo/odoo/pull/153440 (which targets 15.0)

**Pull request closed without merging**